deploy/role.yaml

```diff
@@ -5,6 +5,12 @@ metadata:
   creationTimestamp: null
   name: tower-operator
 rules:
+  - apiGroups:
```

Comment:
Will this cause any issues on non-OpenShift clusters?

Reply:
I ran this on a 1.17 KinD cluster and it doesn't appear to cause any issues.